### DISCUSSION TITLE
Is it possible to autoplay audio file on a slide?

### DISCUSSION THREAD

**Discussion by durraniu** [quarto-dev/quarto-cli#11370] [Q&A] [answered]
### Description

I have audio clips for each slide that I want to autoplay when a slide is visited. Is that possible?

**Comment by cderv** [comment, marked as answer]
Revealjs has some support for this : https://revealjs.com/media/#autoplay
It should be working in Quarto generated slides too. The global config is in our reference page too: https://quarto.org/docs/reference/formats/presentations/revealjs.html#media

If this is not working, or you need specific, you should look at the API event for revealjs. There is a slidechanged event (https://revealjs.com/events/#slide-changed) and you can do some JS processing when triggered.

**Reply by durraniu** [reply]
Thanks a lot.

**Reply by durraniu** [reply]
Quick update: It works. For example, I put this in a slide:
```
<audio>
  <source src="path/to/audio_file.wav" type="audio/wav">
</audio> 
```
And put `auto-play-media: true` in revealjs options in the YAML.
